src/app/campaign.model.ts

```diff
@@ -39,5 +39,6 @@ export class Campaign {
     public championRef?: string,
     public parentRef?: string,
     public video?: {provider: string, key: string},
+    public surplusDonationInfo?: string,
```

Comment:
Better to keep optional params alphabetical for readability / maintainability. Can't remember if this one has been reworked yet but we should try to avoid direct constructors that depend on the param order so new param additions anywhere besides the end aren't too painful to make going forward.